app/src/main/java/com/kickstarter/ui/activities/WebViewActivity.java

```diff
@@ -47,12 +46,19 @@ protected void onCreate(final @Nullable Bundle savedInstanceState) {
       .subscribe(webView::loadUrl);
   }
 
-  @OnClick(R.id.back_button)
-  public void backButtonClick() {
-    back();
+  @Override
+  public void back() {
+    if (webView.canGoBack()) {
```

Comment:
from the docs: https://developer.android.com/guide/webapps/webview.html#HandlingNavigation

seems pretty chill to either navigate back through the webview stack, or invoke our `BaseActivity`'s `back()` method which handles general back navigation